.travis.yml

```diff
@@ -8,19 +8,18 @@ install:
 services:
   - postgresql
 before_script:
-  - psql -c "CREATE DATABASE testAuthorsHaven;" -U postgres
+  - psql -c "CREATE DATABASE test;" -U postgres
 script:
   - coverage erase
-  - coverage run --source='.' manage.py test
+  - coverage run --source=authors/apps manage.py test
   - coverage report -m
   - coveralls
 after_success:
   - coveralls
 
-deploy:
+? deploy
```

Comment:
@meshack-mbuvi what's happening here?

Reply:
Whenever I save the file, the editor is turning that statement to look as you have seen above.